omero/developers/Modules/Search.rst

```diff
@@ -57,10 +61,20 @@ FullTextAnalyzer <src/main/java/ome/services/fulltext/FullTextAnalyzer.java>`.
 
 Assuming these entries above for Image.name:
 
--  searching for **GFP-H2B** returns 1 and 2.
--  searching for **"GFP H2B"** also returns 1 and 2.
--  searching for **GFP H2B** returns 1, 2, and 3, since the two terms
+-  searching for **GFP-H2B** returns 1, 2, 3 and 4, because of the tokenizing on the hyphen and joining of the tokens by an **OR**.
+-  searching for **"GFP H2B"** only returns 1 and 2, since the quotes enforce the exact sequence of the tokens and the query is built with an **AND**.
+-  searching for **GFP H2B** returns 1, 2, 3 and 4, since the two tokens
    are joined by an **OR**.
+-  searching for **"GFP-H2B"** returns 1 and 2.
```

Comment:
This could be combined with the second example, no? So does that mean that the hyphen is just always factored out?

Reply:
Will do. Yes, in all my experiments this holds.